src/overlays/actors/ovl_Bg_Dblue_Movebg/z_bg_dblue_movebg.c

```diff
@@ -28,46 +59,829 @@ const ActorInit Bg_Dblue_Movebg_InitVars = {
     (ActorFunc)BgDblueMovebg_Draw,
 };
 
-#endif
+static Gfx* D_80A2B8AC[] = {
+    NULL,
+    object_dblue_object_DL_0069D8,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    object_dblue_object_DL_004848,
+    object_dblue_object_DL_0061B8,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+};
+
+static s32 D_80A2B8DC[] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0x0600CAA0, 0 };
+
+static CollisionHeader* D_80A2B90C[] = {
+    NULL,
+    &object_dblue_object_Colheader_006EA8,
+    NULL,
+    NULL,
+    NULL,
+    &object_dblue_object_Colheader_00D3DC,
+    &object_dblue_object_Colheader_005D28,
+    &object_dblue_object_Colheader_00714C,
+    &object_dblue_object_Colheader_00AED0,
+    &object_dblue_object_Colheader_00AED0,
+    NULL,
+    NULL,
+};
+
+static TexturePtr D_80A2B93C[] = {
+    NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, &object_dblue_object_Matanimheader_00CC18, NULL,
+};
```

Comment:
An AnimatedMaterial isn't really a texture, so `TexturePtr` is the wrong type here.
```suggestion
static AnimatedMaterial D_80A2B93C[] = {
    NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, &object_dblue_object_Matanimheader_00CC18, NULL,
};
```